lib/validator.js

```diff
@@ -82,6 +82,9 @@ export default class Validator {
 
         const run = (v, rules, triedToConvert = false) => {
             const rule = rules[0];
+            if (v === undefined && rule._default) {
+                v = rule._default;
+            }
```

Comment:
I'd rather do that as long we don't need this functionality.

I added a reminder to refactor this later when we add more related functionality.